source/Core/Main.cpp

```diff
@@ -27,6 +34,13 @@ namespace SuperHaxagon {
 		return std::make_unique<PlatformLinux>(Dbg::INFO);
 		#elif defined _nspire
 		return std::make_unique<PlatformNspire>(Dbg::INFO);
+		#elif defined __APPLE__
+			#include "TargetConditionals.h"
```

Comment:
Does "TargetConditionals.h" need to be re-included here?

Reply:
It is used to resolve `TARGET_OS_MAC`.

My intent was to exclude iOS devices since I did not test the build on those devices.
But double checking my code, and according to https://stackoverflow.com/questions/12132933/preprocessor-macro-for-os-x-targets, it seems this is not enough.

instead of `TARGET_OS_MAC` that should be `TARGET_OS_OSX`.
I did the change already.

Reply:
Oh, to me it looks like it's being included twice. Was just wondering if I can get rid of the second include and not have things break. I don't have a mac to otherwise test the changes and see if it builds.

Reply:
I'll just merge this as-is, thanks for adding support!

Reply:
Oops sorry, it seams I never sent the answer I wrote... Here it is, albeit a bit late:

> Indeed you are right, I didn't pay attention to the fact I was already including the same file some lines above.
> I tested it and obviously that second occurrence can be removed.
> I did the change already.

But still I'm going nuts, I did not push the change either... must have been tired when I did it :( !